test/clients/binary_framed_test.exs

```diff
@@ -0,0 +1,63 @@
+defmodule BinaryFramedTest do
+  use ThriftTestCase
+
+  alias Thrift.Clients.BinaryFramed
+
+  @thrift_file name: "void_return.thrift", contents: """
+  service VoidReturns {
+    void my_call(1: i64 id)
+  }
+  """
+  alias BinaryFramedTest.VoidReturns
+  alias Thrift.TApplicationException, as: TAE
```

Comment:
I would have thought VoidReturns had already been aliased by ThriftTestCase.

All the aliases should be grouped together up at the top. They don't need to be below the `@thrift_file`.